{"identifier":{"url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"metadata":{"modules":[{"name":"Auth0"}],"role":"collectionGroup","title":"LocalizedError Implementations","roleHeading":"API Collection"},"schemaVersion":{"minor":3,"major":0,"patch":0},"topicSections":[{"generated":true,"identifiers":["doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/failureReason","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/helpAnchor","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/recoverySuggestion"],"title":"Instance Properties"}],"sections":[],"hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError"]]},"kind":"article","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/auth0\/authenticationerror\/localizederror-implementations"]}],"references":{"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/recoverySuggestion":{"type":"topic","abstract":[],"url":"\/documentation\/auth0\/authenticationerror\/recoverysuggestion","title":"recoverySuggestion","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"recoverySuggestion","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/recoverySuggestion"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/helpAnchor":{"title":"helpAnchor","role":"symbol","url":"\/documentation\/auth0\/authenticationerror\/helpanchor","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/helpAnchor","abstract":[],"type":"topic","kind":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}]},"doc://com.auth0.Auth0/documentation/Auth0":{"kind":"symbol","url":"\/documentation\/auth0","role":"collection","title":"Auth0","type":"topic","abstract":[{"text":"SDK for Apple platforms.","type":"text"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/failureReason":{"title":"failureReason","role":"symbol","abstract":[],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/failureReason","url":"\/documentation\/auth0\/authenticationerror\/failurereason","type":"topic","kind":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}]},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError":{"fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"kind":"identifier","text":"AuthenticationError"}],"role":"symbol","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"AuthenticationError"}],"title":"AuthenticationError","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError","url":"\/documentation\/auth0\/authenticationerror","abstract":[{"type":"text","text":"Represents an error during a request to the Auth0 Authentication API."}]}}}